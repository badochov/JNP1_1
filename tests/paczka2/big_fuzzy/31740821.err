Error in line 4: HQpHGrUO9w S803 32,9
Error in line 14: Ck0 S58 20,0
Error in line 26: HQpHGrUO9w S58 -2,9
Error in line 18: EsGmaVdu S803 5,6
Error in line 29: EsGmaVdu A96 33,7
Error in line 25: HQpHGrUO9w S58 21,4
Error in line 47: A661 A96 9,0
Error in line 54: MHxvG S803 40,3
Error in line 51: hp6X4 S58 28,1
Error in line 49: HQpHGrUO9w S803 65,4
Error in line 89: Ck0 S803 10,9
Error in line 98: EsGmaVdu S803 349,3
Error in line 107: EsGmaVdu A661 106,6
Error in line 93: Ve4fH S58 280,6
Error in line 101: MHxvG S58 36,8
Error in line 113: Ve4fH A96 34,3
Error in line 122: hp6X4 S58 44,7
Error in line 128: MHxvG S803 18,20
Error in line 125: A661 A661 85,8
Error in line 133: Ve4fH A96 75,5
Error in line 139: MHxvG S803 23,1
Error in line 148: MHxvG S58 -2,6
Error in line 152: EsGmaVdu A661 51,9
Error in line 136: Ck0 A96 51,8
Error in line 156: A661 S803 1,0
Error in line 155: Ck0 S58 37,8
Error in line 143: HQpHGrUO9w S803 60,0
Error in line 163: Ck0 S803 -11,8
Error in line 160: Ck0 S803 4,8
Error in line 161: HQpHGrUO9w S58 80,8
Error in line 177: EsGmaVdu A96 59,2
Error in line 182: EsGmaVdu A661 3,9
Error in line 186: hp6X4 A661 87,7
Error in line 179: MHxvG S58 102,8
Error in line 191: HQpHGrUO9w A96 -1,5
Error in line 195: EsGmaVdu S803 90,0
Error in line 199: EsGmaVdu S58 30,7
Error in line 214: A661 A661 44,8
Error in line 215: A661 A96 89,2
Error in line 229: Ve4fH A96 44,9
Error in line 227: HQpHGrUO9w A96 172,2
Error in line 233: Ve4fH A661 38,6
Error in line 242: hp6X4 A661 22,40
Error in line 209: hp6X4 S58 73,8
