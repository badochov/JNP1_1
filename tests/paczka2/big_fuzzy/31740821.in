?
Ve4fH S803 36,0
hp6X4 A661 65,2
HQpHGrUO9w S803 32,9

hp6X4 A661 295,4

Ve4fH S803 33,5
?HQpHGrUO9w
HQpHGrUO9w S58 31,8
HQpHGrUO9w S58 58,2
Ve4fH A96 78,4
hp6X4 S58 19,1
Ck0 S58 20,0
hp6X4 S58 6,3
Ck0 A661 98,3
A661 S803 19,6
EsGmaVdu S803 5,6
Ve4fH A96 385,7
?Ck0


Ve4fH A661 4,1
MHxvG A661 57,4
HQpHGrUO9w S58 21,4
HQpHGrUO9w S58 -2,9
MHxvG A661 70,6
Ve4fH A661 148,9
EsGmaVdu A96 33,7
MHxvG S803 15,2
?A661
?S58
MHxvG S803 99,4
EsGmaVdu S58 16,7

A661 S803 9,4

HQpHGrUO9w S803 53,8
?
MHxvG S803 97,4
Ck0 A661 83,6
HQpHGrUO9w S803 252,3
Ve4fH S803 34,4
MHxvG S803 13,7
EsGmaVdu S58 70,0
?Ck0
A661 A96 9,0
A661 S58 95,7
HQpHGrUO9w S803 65,4
Ve4fH S803 118,7
hp6X4 S58 28,1
?A661
A661 S58 29,7
MHxvG S803 40,3
A661 S58 9,0
?
?HQpHGrUO9w
A661 S58 11,6
MHxvG A96 72,1

MHxvG A96 201,2
hp6X4 A661 79,3
QpGrUO9w S803 198,2
EsGmaVdu S803 54,4
EsGmaVdu S803 131,1
A661 S58 85,0
Ck0 A96 68,3
MHxvG A661 86,2
MHxvG A661 142,2
hp6X4 A661 0,0
A661 S58 108,8
hp6X4 S58 40,6
hp6X4 S58 46,2
MHxvG A661 62,1
Ve4fH S58 35,4
hp6X4 S58 122,9
Ck0 A96 195,3
MHxvG A661 287,5
EsGmaVdu S803 45,9
EsGmaVdu S803 126,2
hp6X4 S58 23,9
Ve4fH S58 91,7
?S803
HQpHGrUO9w A661 93,8
EsGmaVdu A96 68,2
EsGmaVdu A96 248,9
?MHxvG
HQpHGrUO9w A661 1,9
Ck0 S803 10,9
Ck0 S58 87,2
Ck0 S58 419,4
EsGmaVdu S803 71,2
Ve4fH S58 280,6
EsGmaVdu S803 97,3
Ck0 S803 67,5
MHxvG S803 75,6
MHxvG S803 227,6
EsGmaVdu S803 349,3
EsGmaVdu A661 28,3
A661 A661 59,3
MHxvG S58 36,8
Ck0 S803 77,6
A661 A661 164,9
EsGmaVdu A661 73,5
HQpHGrUO9w A96 73,5
?Ck0
EsGmaVdu A661 106,6
Ck0 A661 80,4
A661 A661 96,9
Ck0 A661 58,7
HQpHGrUO9w A96 131,6
EsGmaVdu A96 61,4
Ve4fH A96 34,3
MHxvG A96 55,4
?
EsGmaVdu A96 275,1
Ve4fH S58 89,2
Ve4fH S58 372,9
Ck0 S58 43,7
?hp6X4
MHxvG A96 268,7
hp6X4 S58 44,7
EsGmaVdu A96 93,3
A661 A661 398,3
A661 A661 85,8
EsGmaVdu A96 386,3
hp6X4 S803 4,2
MHxvG S803 18,20
A661 S58 91,7

A661 S58 255,8
Ck0 S58 118,0
Ve4fH A96 75,5
A661 A96 7,2
hp6X4 S803 2,5
Ck0 A96 51,8
A661 A96 30,6
HQpHGrUO9w S803 10,4
MHxvG S803 23,1
hp6X4 S803 43,7
Ve4fH S803 44,0
HQpHGrUO9w S803 24,1
HQpHGrUO9w S803 60,0
MHxvG S58 34,8
hp6X4 S803 166,9
hp6X4 S803 21,2
Ve4fH S803 220,0
MHxvG S58 -2,6
A661 A661 81,7
hp6X4 S803 22,9

EsGmaVdu A661 51,9
EsGmaVdu S803 38,4
A661 A661 187,9
Ck0 S58 37,8
A661 S803 1,0

A661 S58 56,8
Ve4fH A661 77,1
Ck0 S803 4,8
HQpHGrUO9w S58 80,8
EsGmaVdu S803 99,5
Ck0 S803 -11,8

Ck0 S58 72,2
A661 S58 211,9
EsGmaVdu A96 31,8
hp6X4 A96 22,6
Ve4fH A661 797,4

EsGmaVdu A96 31,8
HQpHGrUO9w A96 47,9
?HQpHGrUO9w
Ck0 S58 156,0
MHxvG S58 50,9
Ve4fH A96 56,0
EsGmaVdu A96 59,2
EsGmaVdu A661 63,9
MHxvG S58 102,8
EsGmaVdu A661 64,2
hp6X4 A96 101,9
EsGmaVdu A661 3,9
EsGmaVdu S58 36,4
?S803
Ck0 A661 88,1
hp6X4 A661 87,7
Ck0 A661 207,9
hp6X4 A96 97,2
EsGmaVdu S58 16,0
MHxvG A661 42,6
HQpHGrUO9w A96 -1,5
Ve4fH A96 155,7
MHxvG A661 195,5
?Ck0
EsGmaVdu S803 90,0
?S803
hp6X4 A96 2,9
Ve4fH S803 35,7
EsGmaVdu S58 30,7
A661 S803 46,7
hp6X4 A96 30,0
MHxvG S803 83,4
Ck0 S58 35,2
hp6X4 A96 49,8
Ve4fH S803 81,8
EsGmaVdu S803 8,1
A661 S803 148,5
?A96
hp6X4 S58 73,8
EsGmaVdu S803 34,2
?A96
Ck0 S58 113,0
MHxvG S803 112,6
A661 A661 44,8
A661 A96 89,2
HQpHGrUO9w A96 41,3
A661 A661 50,4
MHxvG A96 90,4
EsGmaVdu A661 46,9
?hp6X4
EsGmaVdu A661 23,9
MHxvG A96 183,5
?
A661 A661 81,6
Ck0 A96 356,5
EsGmaVdu S803 91,3
HQpHGrUO9w A96 172,2
Ck0 A96 80,2
Ve4fH A96 44,9
EsGmaVdu S803 272,1
EsGmaVdu A96 79,5
EsGmaVdu A96 379,2
Ve4fH A661 38,6

HQpHGrUO9w S803 9,2
A661 A96 21,4
EsGmaVdu S803 0,0
A661 A96 61,8
EsGmaVdu S803 0,0
Ck0 A661 34,3
Ve4fH S803 56,0
hp6X4 A661 22,40
hp6X4 A661 39,4
?A661
EsGmaVdu S803 40,3
HQpHGrUO9w S803 393,4
EsGmaVdu S803 97,2
A661 A96 28,6
?
A661 A96 90,1