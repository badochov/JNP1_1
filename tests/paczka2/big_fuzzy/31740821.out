A661 388,2
S58 39,2
A661 S 10,2
HQpHGrUO9w S 26,4
MHxvG A 13,2 S 84,2
Ve4fH A 452,1 S 2,5
hp6X4 A 230,2 S 12,8
S58 39,2
A96 307,3
A661 388,2
S803 96,9
Ck0 A 14,7
A661 S 10,2
A661 402,9
A661 S 76,2
Ck0 A 14,7
EsGmaVdu S 53,3
HQpHGrUO9w S 224,9
MHxvG A 13,2 S 167,9
Ve4fH A 452,1 S 86,8
hp6X4 A 230,2 S 12,8
S58 158,5
A96 307,3
A661 402,9
S803 463,4
HQpHGrUO9w S 224,9
S803 620,4
MHxvG A 423,7 S 167,9
Ck0 A 141,7 S 342,3
A661 A 105,6 S 102,6
Ck0 A 163,4 S 342,3
EsGmaVdu A 225,9 S 236,4
HQpHGrUO9w A 150,0 S 224,9
MHxvG A 423,7 S 319,9
Ve4fH A 452,1 S 143,1
hp6X4 A 309,5 S 117,4
S58 678,0
A96 802,2
A661 1028,0
S803 808,6
hp6X4 A 309,5 S 117,4
HQpHGrUO9w A 150,0 S 238,6
S803 1186,0
Ck0 A 283,2 S 500,4
S803 1186,0
A96 1838,7
A96 1838,7
hp6X4 A 502,9 S 244,0
A661 A 536,6 S 523,6
Ck0 A 283,2 S 578,2
EsGmaVdu A 755,9 S 344,0
HQpHGrUO9w A 156,6 S 238,6
MHxvG A 883,0 S 365,2
Ve4fH A 1272,1 S 648,9
hp6X4 A 502,9 S 244,0
S58 1553,3
A96 1938,4
A661 2451,9
S803 1389,2
A661 A 608,2 S 523,6
A661 2483,1
A661 A 608,2 S 523,6
Ck0 A 559,5 S 578,2
EsGmaVdu A 1055,6 S 581,7
HQpHGrUO9w A 156,6 S 622,8
MHxvG A 883,0 S 365,2
Ve4fH A 1272,1 S 648,9
hp6X4 A 502,9 S 244,0
S58 1553,3
A96 2554,8
A661 2483,1
S803 2011,1
